cli/tests/test_config_command.rs

```diff
@@ -628,14 +628,20 @@ fn test_config_get() {
     "###);
 
     let stdout = test_env.jj_cmd_failure(test_env.env_root(), &["config", "get", "table.list"]);
+    #[cfg(unix)]
     insta::assert_snapshot!(stdout, @r###"
-    Config error: invalid type: sequence, expected a value convertible to a string
+    Config error: invalid type: sequence, expected a value convertible to a string for key `table.list` in config/config0002.toml
+    For help, see https://github.com/martinvonz/jj/blob/main/docs/config.md.
+    "###);
+    #[cfg(windows)]
+    insta::assert_snapshot!(stdout, @r###"
```

Comment:
better to normalize the output by `.replace()` or `insta::with_settings!({filters => ..})`. It would be annoying if we had to run tests on both platforms to update the snapshot.

Reply:
Good call, I added a filter.